spec/requests/conversion_hosts_spec.rb

```diff
@@ -1,7 +1,11 @@
 describe "ConversionHosts API" do
+  before(:all) do
```

Comment:
@bdunne Do we still discourage `before(:all)`

Reply:
Only needs to be run once, right? Otherwise it's a peformance hit, isn't it?

Reply:
`before(:all)` has side effects because of how it behaves with respect to database rollbacks after each test.  We generally avoid it because it can be a source of tests leaking into each other.